library/core/src/alloc/layout.rs

```diff
@@ -492,3 +500,24 @@ impl fmt::Display for LayoutError {
         f.write_str("invalid parameters to Layout::from_size_align")
     }
 }
+
+#[cfg(kani)]
+#[unstable(feature="kani", issue="none")]
+mod verify {
+    use super::*;
+    use crate::kani;
+
+    #[kani::proof_for_contract(from_size_align_unchecked)]
+    pub fn check_from_size_align_unchecked() {
+        let s = kani::any::<usize>();
+        let shift_index = kani::any::<u8>();
+        let a : usize = 1 << shift_index;
+
+        kani::assume(a > 0);
+        // kani::assume(s <= isize::MAX as usize - (a - 1));
+
+        let layout = from_size_align_unchecked(s, a);
```

Comment:
These 4000+ errors are very overwhelming. Need to figure out a way to fix this. Anyway, it seems that the compiler gets a bit confused when some compilation stage fails (it doesn't always happen), I'm starting to think this happens at the name resolution stage.

But I believe this is the relevant error:
```
2024-07-09T21:17:54.5064710Z error[E0425]: cannot find function `from_size_align_unchecked` in this scope
2024-07-09T21:17:54.5065410Z    --> /Users/runner/work/verify-rust-std/verify-rust-std/verify-rust-std/library/core/src/alloc/layout.rs:519:22
2024-07-09T21:17:54.5065900Z     |
2024-07-09T21:17:54.5066140Z 510 |     #[kani::proof_for_contract(from_size_align_unchecked)]
2024-07-09T21:17:54.5066720Z     |     ------------------------------------------------------ similarly named function `check_from_size_align_unchecked` defined here
2024-07-09T21:17:54.5067140Z ...
2024-07-09T21:17:54.5067350Z 519 |         let layout = from_size_align_unchecked(s, a);
2024-07-09T21:17:54.5067840Z     |                      ^^^^^^^^^^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `check_from_size_align_unchecked`
```

The problem here is that you are missing the type when invoking `from_size_align_unchecked`, i.e.: `Layout::from_size_align_unchecked`.

Reply:
You will be right. Fixed in the most recent commit.